lib/util.js

```diff
@@ -39,4 +39,11 @@ let getChannel = (channel) => {
   return (process.env.npm_config_test !== undefined) ? TEST_ID : channel;
 }
 
-module.exports = { catchErrors, setRandomNowPlaying, getChannel }
\ No newline at end of file
+// Checks that the user is sending a command in the #bot-stuff channel
+let inCommandChannel = (msg) => {
```

Comment:
the reason why I have the `getChannel` method was because I wanted to test it in an admin-only channel when i'm running it locally instead of a public one, so that everyone does not get spammed by my test commands. Can you use `getChannel` as part of `inCommandChannel`? 

Also, `msg.channel.id === BOT_STUFF_ID` already returns a boolean value so you could do something like `return msg.channel.id === BOT_STUFF_ID`